yellowbrick/classifier/confusion_matrix.py

```diff
@@ -80,6 +84,7 @@ def __init__(self, model, ax=None, classes=None, **kwargs):
         self.cmap.set_under(color='w')
         self.cmap.set_over(color='#2a7d4f')
         self.edgecolors=[] #used to draw diagonal line for predicted class = true class
+        self.label_mapping= label_mapping
```

Comment:
This is an extremely interesting idea, and I like what you've done. My question though is that I wonder if the user will actually have a mapping like the one you suggest. Later on you use the `LabelEncoder` for guessing the mapping between classes and numeric values, but likely that's what the user used to do the encoding (unless they used something like `pandas.get_dummies`). What would you think of having the user optionally pass in their label encoder, rather than a dictionary?